.gitignore

```diff
@@ -1,6 +1,7 @@
 *.swp
 .DS_Store
 .cache
+**/tmp/
```

Comment:
This is to cover any spark directories or other java tmp files people might generate inside the repo during the course of working on one of the many projects we support. It was originally underneath /spark so I assume that a test probably generates it. It will still be ignored this way.